### ISSUE TITLE
Ability to use the new animation movement also for npcs

### ISSUE THREAD

**Issue by alexderpyfox** [Yellow-Dog-Man/Resonite-Issues#3099]
### Is your feature request related to a problem? Please describe.

It no longer does movement animation when not wearing the avatar while with the previous system it worked but now it just slides around so the ability to use the new moving animation with npcs would be nice 

### Describe the solution you'd like

It be able to be used for npcs 

### Describe alternatives you've considered

Mods

### Additional Context

Npc moving walking Animation

**Comment by alexderpyfox** [comment]
Npcs are already made using protoflux for movement only the walking isn't there which first was also protoflux and then it was the ik and now there is the new movement animation which can't be used

**Comment by alexderpyfox** [comment]
It was not thought about that people will use the movement animation also for npcs in the discord the discussion was there as well about what is a feature and what is unintended use because it worked fine without needing to do something complex or "refhacking" before so it feels like a overseen feature

**Comment by alexderpyfox** [comment]
Or add the ability to use animation clips like unity has with an animation controller

**Comment by troyBORG** [comment]
Yeah I swear there was something back in neos that you'd place an avatar somewhere for it to assign it, then you click record you run around even in full body and record the animation.   Then click stop.  Then the avatar you assigned would repeat that animation.

**Comment by Dusty-Sprinkles** [comment]
> Yeah I swear there was something back in neos that you'd place an avatar somewhere for it to assign it, then you click record you run around even in full body and record the animation. Then click stop. Then the avatar you assigned would repeat that animation.

I know of one of those mocap systems, though for this I think setting up multiple may be slightly cumbersome (one of which can be found at resrec:///U-guheheP/R-00ad625c-7401-4e58-8ae6-f2dfdca59b89 though I believe there are others now)

If its not a huge workaround/bad temporary fix maybe some kind of toggle/option could be added to allow the locomotion to be simulated without a user preset since I imagine there are some slight performance gains for non-"npc" avatars not using the animation while idle

**Comment by AmasterAmaster** [comment]
It would be cool to have NPC support of some kind.

**Comment by alexderpyfox** [comment]
how do i add a community help welcome tag to this?

**Comment by Frooxius** [comment]
You don't. We don't have any tasks on this that community could help with right now.
